src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -297,6 +297,9 @@
 
             @Config.Comment("Shader Settings")
             public Shader shader = new Shader();
+
+            @Config.Comment("Allows player data to be stored in .minecraft for cross-modpack purposes, such as capes. Default: true")
+            public boolean persistentDataActive = true;
```

Comment:
I would say don't make this a config option, and just do it

Reply:
Actually, given that I removed any use of this... I should probably just yeet the config anyways.